src/apd.go

```diff
@@ -0,0 +1,108 @@
+package src
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"time"
+)
+
+const (
+	defaultBroadCastIP = "255.255.255.255"
+)
+
+type Packet struct {
+	PublicKey string
+	IP        string
+}
+
+type Apd struct {
```

Comment:
I would name it `APD` (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)